apps/avatax/integration/data/templates/checkout.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Checkout:USA": {
+    "channelSlug": "$M{Channel.USA.slug}",
+    "variantId": "$M{Product.Regular.variantId}",
+    "email": "$F{UserEmail}",
+    "address": "$M{Address.NewYork}"
```

Comment:
I like this templates approach (at least reading it) → reduces some boilerplate, make it easier to read and also lowers the bar for more business people to understand whats going on